fair-api-datasets-get.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import json
+import requests
+from common.auth import AUTHENTICATED_HEADERS
+from common.constants import DATASETS_URL, SSL_VERIFY
+
+if len(sys.argv) != 2:
+    print(f'Usage: {sys.argv[0]} <code>')
+    exit(1)
+code = sys.argv[1]
+dataset_url = f"{DATASETS_URL}{code}"
+response = requests.get(
+    dataset_url, headers=AUTHENTICATED_HEADERS, verify=SSL_VERIFY)
+if response.status_code != 200:
+    error_data = response.json()
+    print(
+        f'Failed to retrieve Dataset. Status code: {response.status_code}, Error message: {error_data["error"]["message"]}')
+    exit(2)
```

Comment:
Think this should be an exit status 1

Reply:
I made it exit status 2 because the convention in other files is to use exit status 1 for missing arguments, token, or endpoint. Exit status 2 means that a response failed. I can make this more explicit.